main/test/flix/Test.Exp.Regions.flix

```diff
@@ -556,7 +556,10 @@ namespace Test/Exp/Regions {
                 MutList.push!([1; 8] @ r2, l);
                 MutMap.put!(1, l, m);
                 match MutMap.get(1, m) {
-                    case Some(k) => (MutList.get(0, k))[0] == 1
```

Comment:
This feels too messy to me. Can we do: `MutList.nth(0, k) |> Option.flatMap(Array.nth(0)) == Some(1)`?

Or alternatively add a helper function? 

(Applies to the rest of the test cases)